__tests__/api-test.test.js

```diff
@@ -164,6 +164,72 @@ describe("/api/articles/:article_id/comments", () => {
         expect(msg).toBe('article does not exist');
       });
 
+  });  
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH 200: increments the votes property on article id by the vote amount provided on the request", () => {
+    const newVotes = {
+      inc_votes: 20
+    };
+    return request(app)
+      .patch('/api/articles/1')
+      .send(newVotes)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveProperty("title", "Living in the shadow of a great man");
+        expect(article).toHaveProperty("author", "butter_bridge");
+        expect(article).toHaveProperty("article_id", 1);
+        expect(article).toHaveProperty("topic", "mitch");
+        expect(article).toHaveProperty("created_at", '2020-07-09T20:11:00.000Z');
+        expect(article).toHaveProperty("votes", 120);
+        expect(article).toHaveProperty("article_img_url", "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
+      });
   });
-  
+
+  test("PATCH 200: decrements the votes property on article id by the vote amount provided on the request", () => {
+    const newVotes = {
+      inc_votes: -30
+    };
+    return request(app)
+      .patch('/api/articles/1')
+      .send(newVotes)
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toHaveProperty("votes", 70);
+      });
+  });
+
+  test("PATCH 400: responds with error message when provided using an invalid id", () => {
+    const newVotes = {
+      inc_votes: -30,
+    };
+    return request(app)
+      .patch("/api/articles/999")
+      .send(newVotes)
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("article does not exist");
+      });
+  });
+
+  test("PATCH 400: responds with error message when no message is provided", () => {
+    const newVotes = {
```

Comment:
It makes sense that you have added a 400 here as the user is not providing us with the info we expect, however, if we think about this from a behavioural point of view, if nothing is passed through then the data returns unchanged. In that case, this should in fact probably be a 200. Furthermore, that would allow us to expand on this in the future if we wanted to, for example, expand the ability to patch for things like the body of the article, etc without the need to vote every time.